examples/air-quality-sensor-app/silabs/include/AppEvent.h

```diff
@@ -19,6 +19,9 @@
 
 #pragma once
 #include <cstdint>
+#ifdef DISPLAY_ENABLED
+#include "lcd.h"
+#endif
 
 struct AppEvent;
```

Comment:
This change in AppEvent.h for all apps makes me realize that there are duplications and that we should have a base AppEvent Struct. Each app should inherit the base AppEvent and add the particularity after the inheritance.

I'm not sure how/where this would be defined but we could reduce this duplication in all apps headers

Reply:
yes, I have already planned to add this in CSA, since it is just a patch release don't want to add any new files. I can add the same changes here as well if needed.